libsolidity/interface/CompilerStack.cpp

```diff
@@ -1024,6 +1025,37 @@ Json::Value CompilerStack::methodIdentifiers(string const& _contractName) const
 	return methodIdentifiers;
 }
 
+Json::Value CompilerStack::errorIdentifiers(string const& _contractName) const
```

Comment:
I think we should change `methodIdentifiers` to return a single large JSON and display them that way on the CLI, and pass it down as a new field in standard json.

Reply:
Good point (I'd rename it from `methodIdentifiers()` to something more genric though).

Just to make things clear - you don't mean literally printing the JSON on the CLI but just passing that JSON to `CommandLineInterface` and then printing it the same way we do now, right?

Also, what's your opinion on https://github.com/ethereum/solidity/pull/12548#pullrequestreview-855403715? Does it make sense to add it to `--combined-json` or should we keep it as is since it's going to be removed anyway?

> pass it down as a new field in standard json.

I guess it would be fine to do it in a separate PR. I don't think we have this info in Standard JSON even for method identifiers right now.

Reply:
> Just to make things clear - you don't mean literally printing the JSON on the CLI but just passing that JSON to CommandLineInterface and then printing it the same way we do now, right?

No, it still needs to be displayed nicely, but that can be done by iteration the two-level JSON.

> Does it make sense to add it to --combined-json or should we keep it as is since it's going to be removed anyway?

Please do not touch `--combined-json`, it is due to be removed next breaking release.

>  I don't think we have this info in Standard JSON even for method identifiers right now.

I think we have it.

Reply:
> I think we have it.

Right, we do. It's `evm.methodIdentifiers`. Not sure how I missed that.

In that case @Marenz I think we should add `evm.eventIdentifiers` and `evm.errorIdentifiers` too. Either in this PR or in a separate one (either is fine with me).

Reply:
> I think we should change `methodIdentifiers` to return a single large JSON and display them that way on the CLI, and pass it down as a new field in standard json.

What do you think of this one @Marenz?